Ext4Pkg/Ext4Dxe/Ext4Dxe.c

```diff
@@ -260,10 +260,12 @@ Ext4Stop (
 
 EFI_DRIVER_BINDING_PROTOCOL  gExt4BindingProtocol =
 {
-  Ext4IsBindingSupported,
-  Ext4Bind,
-  Ext4Stop,
-  EXT4_DRIVER_VERSION
+  .Supported           = Ext4IsBindingSupported,
+  .Start               = Ext4Bind,
+  .Stop                = Ext4Stop,
+  .Version             = EXT4_DRIVER_VERSION,
+  .ImageHandle         = NULL,

```

Comment:
Per C89: _If an object that has static storage duration is not initialized explicitly, it is initialized implicitly as if every member that has arithmetic type were assigned 0 and every member that has pointer type were assigned a null pointer constant._

TL;DR This commit is useless

Reply:
Otherwise It produces warning in release-build which we treat as error

Reply:
what's the compiler again? MSVC?

Reply:
It is much easier to read the code when initialisers are named explicitly. I believe this commit substantially improves readability.

Reply:
> what's the compiler again? MSVC?

GCC5/CLANGPDB/CLANGDWARF/MSVC throw a warning about DriverBinding protocol fields.
```
../../UDK/Ext4Pkg/Ext4Dxe/Ext4Dxe.c:267:1: error: missing initializer for field ‘ImageHandle’ of ‘EFI_DRIVER_BINDING_PROTOCOL’ {aka ‘struct _EFI_DRIVER_BINDING_PROTOCOL’} [-Werror=missing-field-initializers]
  267 | };
      | ^
In file included from ../../UDK/MdePkg/Include/Library/UefiLib.h:26,
                 from ./../../User/Include/UserPcd.h:38,
                 from <command-line>:
../../UDK/MdePkg/Include/Protocol/DriverBinding.h:180:17: note: ‘ImageHandle’ declared here
  180 |   EFI_HANDLE    ImageHandle;
      |                 ^~~~~~~~~~~
cc1: all warnings being treated as errors
```